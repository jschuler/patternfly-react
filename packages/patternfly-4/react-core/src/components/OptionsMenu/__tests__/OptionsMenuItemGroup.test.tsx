// Auto-generated do not edit

import * as React from 'react';
import { shallow } from 'enzyme';
import { OptionsMenuItemGroup } from '../OptionsMenuItemGroup';

describe('OptionsMenuItemGroup test', () => {
  it('OptionsMenuItemGroup should match snapshot', () => {
    const view = shallow(
      <OptionsMenuItemGroup
        children={<>ReactNode</>}
				className={"''"}
				ariaLabel={"''"}
				groupTitle={''}
				hasSeparator={false}
      />);
    expect(view).toMatchSnapshot();
  });
});
