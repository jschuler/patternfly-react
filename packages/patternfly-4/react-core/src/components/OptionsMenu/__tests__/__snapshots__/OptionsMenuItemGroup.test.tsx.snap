// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OptionsMenuItemGroup test OptionsMenuItemGroup should match snapshot 1`] = `
<section
  className="pf-c-options-menu__group"
>
  <ul
    aria-label="''"
    className="''"
  >
    ReactNode
  </ul>
</section>
`;
